types/react-video-recorder/index.d.ts

```diff
@@ -0,0 +1,88 @@
+// Type definitions for react-video-recorder 3.19
+// Project: https://github.com/fbaiodias/react-video-recorder#readme
+// Definitions by: Gabriel Cartier <https://github.com/GabrielCartier>
+//                 Antonio Herrera <https://github.com/aherrera9>
```

Comment:
```suggestion
// Definitions by: Gabriel Cartier <https://github.com/GabrielCartier>
```
that's not the best idea to incude other person handle. If you want this handle to be used as the owner, explicit OK should be added by this person, as the bot is sending notification to definition owners

Reply:
yea I pinged him but didn't get an answer, didn't think it would notify him in any way. Removed him, thx.